# general options
layer_z_step: 5.0  # unit separation between layers
# ratio of where the split from a single edge to the multiple edges to the mesh occurs.
# 0 -> at the height of the mesh
# 1 -> at the height of the object layer
mesh_edge_break_ratio: 0.5
# where to draw the mesh (0.0 is recommended)
mesh_layer_offset: 0.0
# whether or not to apply offsets to each of the layers
collapse_layers: false
# color places by their distance to the nearest obstalce
color_places_by_distance: false

# places colormap
places_min_distance: 0.0
places_max_distance: 0.5
places_min_hue: 0.6
places_min_saturation: 0.4
places_min_luminance: 0.3
places_max_hue: 0.78
places_max_saturation: 0.9
places_max_luminance: 0.7
